### PR TITLE
[do not merge] add vlc test

### PR DESCRIPTION
**Pull request by nilxam**
do not merge this until Staging:E merged into Factory

### CODE REVIEW

**Comment by coolo** [comment]
as vlc is no longer installed by default - what do you think about having that test on top of installed-kde-image ?

File: tests/x11/vlc.pm [hunk 1]
```diff
@@ -0,0 +1,18 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    # Launch vlc and play a 20 secs video file
+    x11_start_program("vlc ~/data/test_video.ogv");
```
Comment (DimStar77) [inline tests/x11/vlc.pm]: probably we need an ensure_installed here first now..

**Comment by nilxam** [comment]
IIRC the tests what rely on install-kde-image actually never successful due to the desktop widget is disappear after boot from pre-installed kde image. that's why we do not have install-kde-image on openqa.o.o now but install-gnome-image. but we can try again though...

**Comment by coolo** [comment]
so let's kill this ;(

**Comment by lnussel** [comment]
hmm? we do have working kde images
https://openqa.opensuse.org/group_overview/14
https://openqa.opensuse.org/tests/98292

**Comment by coolo** [comment]
to some definition of working - kde is supposed to have a widget on the desktop.
see https://openqa.opensuse.org/tests/97552/modules/first_boot/steps/1

**Comment by lnussel** [comment]
well, the needle for that was added Sep 23 so openqa doesn't consider it an error

**Pull request closed without merging**